.github/workflows/build.yml

```diff
@@ -78,8 +78,9 @@ jobs:
       run: |
         docker run -di --name emscripten -v $(pwd):/src emscripten/emsdk:latest bash
         docker exec emscripten emcc -v
-        docker exec emscripten emcmake cmake .
-        docker exec emscripten make -j 2 VERBOSE=1
+        docker exec emscripten emcmake cmake -B emscripten -DWERROR=ON -DBUILD_TESTS=OFF
+        docker exec -w /src/emscripten emscripten make -j $(nproc)
```

Comment:
Why drop the `VERBOSE=1` here?

Reply:
Just VERBOSE level trace is not quite necessary in most cases, because build warning and error show by default. VERBOSE level trace some time makes warning and error message not quite visible inside a large mount of text.
But it's not a big issue, I can also add it back. What do you think?

Reply:
In think as long as we have `-DWERROR=ON` when its probably find to keep verbose build output when building during CI